Shared/Samples/Show extruded features/ShowExtrudedFeaturesView.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct ShowExtrudedFeaturesView: View {
+    /// A scene with a topographic basemap centered on the US.
+    @State private var scene: ArcGIS.Scene = makeScene()
+    
+    /// The population statistic selection of the picker.
+    @State private var statisticSelection = Statistic.totalPopulation
+    
+    /// The feature layer on the scene.
+    private var featureLayer: FeatureLayer {
+        scene.operationalLayers.last as! FeatureLayer
+    }
+    
+    var body: some View {
+        VStack {
+            SceneView(scene: scene)
+            
+            // Unit system picker.
+            Picker("", selection: $statisticSelection) {
```

Comment:
```suggestion
            Picker("Statistic", selection: $statisticSelection) {
```

Though the `titleKey` doesn't appear to be expressly used at present, it _may_ be used internally for accessibility in some way. I'm not certain either way.

Passing some name though, such as "Statistic" does make the sample code a bit clearer as to what the picker is for and would probably also allow you to remove the inline comment directly above.